pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -137,8 +137,10 @@ func (s *apiScaffolder) updateMainEventRecorder() error {
 }
 
 func (s *apiScaffolder) scafffoldControllerWithImage(scaffold *machinery.Scaffold) error {
+	imageSplit := strings.Split(s.image, ":")
 	controller := &controllers.Controller{ControllerRuntimeVersion: golangv3scaffolds.ControllerRuntimeVersion,
-		Image: s.image,
+		Image:    s.image,
+		ImageTag: imageSplit[1],
```

Comment:
@NikhilSharmaWe, 

We can get this one in the reconcile when so that if the image is changed in the ENV VAR setup we will be able to create the labels accordingly. 

So, can we change this one?

Reply:
Just confirming:
so, we should run the `imageFor` function in the `labelsFor` function for getting the image, and then get the image tag from that value, right?

Reply:
In the labels for after we get the image we can split and get the tag